server/paymentProviders/opencollective/host.js

```diff
@@ -24,7 +26,13 @@ paymentMethodProvider.processOrder = async order => {
   }
 
   const hostFeePercent = await getHostFeePercent(order);
-  const platformFeePercent = await getPlatformFeePercent(order);
+
+  let platformFeePercent;
+  if (order.data.platformTipPercent) {
+    platformFeePercent = order.data.platformTipPercent;
+  } else {
+    platformFeePercent = await getPlatformFeePercent(order);
+  }
```

Comment:
The goal of the `getPlatformFeePercent` utility is to centralize all the complexity. 

I think we should move `data.platformTipPercent` handling is the method itself.

Reply:
Done. 👍🏽